.github/workflows/default.yml

```diff
@@ -137,3 +137,51 @@ jobs:
         with:
           path: dist
           key: ${{ runner.os }}-${{ github.sha }}-dist
+
+  test_e2e:
+    name: E2E
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+
+      - name: Get NPM cache dir
```

Comment:
Gibt es einen Grund die Cache Funktion vom Setup node nicht zu nutzen? https://github.com/actions/setup-node#caching-global-packages-data

Reply:
Danke für den Hinweis. Ich glaube der Grund is lediglich Unwissenheit :D